src/services/pwabuilder.js

```diff
@@ -324,7 +324,7 @@ PWABuilder.prototype.generateImagesForManifest = function(image, manifestInfo, c
 
 PWABuilder.prototype.getServiceWorkerFromURL = function(url) {
   return Q.Promise(async function(resolve,reject){
-    const browser = await puppeteer.launch();
+    const browser = await puppeteer.launch({headless: true, args: ["--no-sandbox"]});
```

Comment:
Just out of curiosity, why are we passing the `no-sandbox` option here?

Reply:
Running chrome/chromium sandboxed inside docker is not supported, whether you're using the root user or not. As of now, this is the only possible configuration to successfully run puppeteer in docker.

As it could be a security issue, we could move this functionality to a separate API in the future.